modules/fluid_structure_interaction/include/interfacekernels/FluidStructureInterface.h

```diff
@@ -0,0 +1,29 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceKernel.h"
+
+class FluidStructureInterface : public InterfaceKernel
+{
+public:
+  static InputParameters validParams();
+
+  FluidStructureInterface(const InputParameters & parameters);
+
+protected:
+  virtual Real computeQpResidual(Moose::DGResidualType type) override;
+  virtual Real computeQpJacobian(Moose::DGJacobianType type) override;
+
+  const MaterialProperty<Real> & _D;
+  const VariableValue & _neighbor_dotdot;
+  const VariableValue & _neighbor_dotdot_du;
+  Real _component;
```

Comment:
Not a `Real`! Use an `unsigned int` for this!

Reply:
Ahhh! How did I miss this! Thank you.